<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SATL Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>811</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabTeams">
     <attribute name="title">
      <string>战队管理</string>
     </attribute>
     <widget class="QListWidget" name="listTeam">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablePlayer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>321</width>
        <height>211</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>昵称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>种族</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addTeam">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delTeam">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPlayer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>360</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delPlayer">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>360</y>
        <width>31</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportTL">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>350</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>导出为TL代码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textTeam">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>战队名称</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editTeam">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editPlayer">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>360</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TLid</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>53</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>种族</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textNickName">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textID">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textRace">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSchedule">
     <attribute name="title">
      <string>排阵管理</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabRestlu">
     <attribute name="title">
      <string>结果管理</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="textCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="changeCode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>切换级别</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
